### DISCUSSION TITLE
Volumes and VolumeMounts in Milvus Service components

### DISCUSSION THREAD

**Discussion by shaiktas** [milvus-io/milvus#33806] [Q&A and General discussion]
Hello Team,
We are deploying milvus using milvus operator approach.  Using this approach once milvus-operator pod is healthy, we using milvus-cluster YAML which is of `kind: Milvus` type and proceeding further for deployment. Being using milvus-cluster YAML if we would like to use volumes and volume Mounts on data node, index nodes what would be the good syntax? I am trying to use below  but no luck with this changes in YAML data node is failing to start. Kindly suggest how to mount any certs on data node, index nodes...
----
    volumes:
      - name: milvus-secret
        secret:
          secretName: milvus-ingress-secret
    dataNode:
      volumeMounts:
        - name: milvus-secret
          readOnly: true
          mountPath: /milvus
-----

**Comment by yhmo** [comment]
Are you trying to set up TLS for milvus? https://milvus.io/docs/tls.md#Set-up-a-Milvus-server-with-TLS
